src/diem/testing/cli/click.py

```diff
@@ -41,6 +41,8 @@ def start_server(
 
 @click.command()
 @click.option("--target", "-t", default="http://localhost:8888", help="Target mini-wallet application URL.")
+@click.option("--host", "-h", default="localhost", help="Miniwallet offchain server host.")
+@click.option("--port", "-p", default=None, help="Miniwallet offchain server port. Random if empty.")
```

Comment:
there are 3 things:
* stub server host, port
* offchain base url

current configuration use localhost as host, find an available port as port, then combine them as offchain base url.

I think we need decouple host and offchain base url here, so we should add:

```
--stub-vasp-base-url
```
it is used to replace default base url

then if we still need customize the host and port, we probably should do:
`--stub-host`, and `--stub-port`